### PR TITLE
Text Improvements

### PR DESCRIPTION
**Pull request by Ebola16**
Fixed a typo and some other improvements

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 ---
 name: Question
-about: Ask a question from the developers
+about: Ask a question for the developers
```
Comment (joepie91) [inline .github/ISSUE_TEMPLATE/question.md]: Maybe "ask the developers a question" instead?

**Comment by nsadeveloper789** [comment]
These look reasonable. I've pulled your branch and submitted it for review by our team.

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (Ebola16) [reply .github/ISSUE_TEMPLATE/question.md]: Sounds more concise, updated.

**Comment by Ebola16** [comment]
Thanks @nsadeveloper789. I updated this branch with joepie91's suggestion.

**Comment by nsadeveloper789** [comment]
OK. Resubmitted.

**Comment by ExtReMLapin** [comment]
When YOU REALLY want to be listed as a contributor

**Comment by Ebola16** [comment]
This PR was made before the full open source release so I went for some low-hanging fruit.

**Pull request merged**